codec.proto

```diff
@@ -13,23 +13,9 @@ message Metadata {
   uint32 schema = 1;
 }
 
-// CommitInfo is basically a replica of tendermint commit info
-// that allows us to store who signed the block in context.
-message CommitInfo {
-  int32 round = 1;
-  repeated VoteInfo votes = 2 [(gogoproto.nullable) = false];
-}
-
-// VoteInfo tells us whether a validator signed the last block or not.
-message VoteInfo {
-  Validator validator = 1 [(gogoproto.nullable) = false];
-  bool signed_last_block = 2;
-}
-
-// Validator represents a validator;
-message Validator {
-  bytes address = 1;
-  int64 power = 3;
+// ValidatorUpdates represents latest validator state, currently used to validate SetValidatorMsg transactions.
+message ValidatorUpdates {
+  repeated weave.ValidatorUpdate validator_updates = 1 [(gogoproto.nullable) = false];
```

Comment:
This declaration results in a structure called `ValidatorUpdates` with a single field `ValidatorUpdates` being a list of `ValidatorUpdate` :roller_coaster: 

I think it would be better to either use different name for the structure, attribute and the attribute type (not `ValidatorUpdate`(s) for everything) or to remove `ValidatorUpdates` container and use `[]weave.ValidatorUpdate` like before.

How about
```
message ValidatorChanges {
    repeated weave.ValidatorUpdate pending = 1 [(gogoproto.nullable) = false];
}
```

Reply:
This is mimicking similar struct in tendermint.

Reply:
In terms of naming - I kind of agree, but this is mostly golang -> protobuf generators fault and ValidatorUpdates describes this best.

It stutters though, so if there is anybody else voting for changing the name - I'll do it. Initially I didn't want to because the container name reflects it's purpose well.

Reply:
We'll have to come up with good names though, current schema matches that of tendermint for a good reason, I don't 100% like it, but `ValidatorChanges -> pending` is going to be even more confusing.

Reply:
Interesting point, lets look where it is used first.

Reply:
Hmmm.... seems like only in memory in the app, and stored in x/validators.

That was the name used there. I'd stick with it, even though I do see the desire for cleaner naming

Reply:
If anything:

```golang
message ValidatorUpdates {
   repeated weave.Validator validators = 1 [(gogoproto.nullable) = false]; 
}
```